cmake/openenclave-config.cmake.in

```diff
@@ -38,6 +50,50 @@ if (UNIX)
       set_target_properties(dl PROPERTIES IMPORTED_LOCATION ${DL_LIB})
     endif ()
   endif ()
+elseif (WIN32)
+  if (NOT NUGET_PACKAGE_PATH)
+    message(FATAL_ERROR "NUGET_PACKAGE_PATH not defined. Please define NUGET_PACKAGE_PATH as the path to the Intel and DCAP Client nuget packages.")
+  endif()
+endif ()
+
+if (NOT TARGET openenclave::sgx_enclave_common)
+  if (UNIX)
+    find_library(SGX_ENCLAVE_COMMON_LIB NAMES sgx_enclave_common)
+  elseif (WIN32)
+    find_library(SGX_ENCLAVE_COMMON_LIB NAMES sgx_enclave_common
+                 PATHS ${NUGET_PACKAGE_PATH}/EnclaveCommonAPI/lib/native/x64-Release)
+  endif ()
+  if (NOT SGX_ENCLAVE_COMMON_LIB)
+    message(FATAL_ERROR "-- Looking for sgx_enclave_common library - not found")
+  else ()
+    message("-- Looking for sgx_enclave_common library - found")
```

Comment:
Any non-fatal error message in the package should probably be `VERBSOE` or `DEBUG`.